src/services/following.service.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  getList,
+  addOne,
+  deleteById,
+} from "../repositories/following.repository";
+
+export const getFollowing = async (queryParams: object) => {
+  const result = await getList(queryParams);
+  return result;
+};
+
+export const addFollowing = async (data: Record<string, string>) => {
```

Comment:
```suggestion
export const addFollowing =  (data: Record<string, string>) => {
```
why did you use async without await?